nuxeo-features/nuxeo-platform-rendition/nuxeo-platform-rendition-core/src/main/java/org/nuxeo/ecm/platform/rendition/lazy/AbstractRenditionBuilderWork.java

```diff
@@ -134,8 +134,9 @@ public void cleanUp(boolean ok, Exception e) {
 
     protected void storeAnErrorRendition() {
         if (session == null) {
-            // The session should be available before cleanup but it depends when the error has been raised
-            openUserSession();
+            log.info("No session, unable to clean work: {} with id: {} for transient store key: {} and document: {}",
```

Comment:
info or warn?

Reply:
We agreed with @kevinleturc to `info` but well...

Reply:
Fair enough